### PR TITLE
Add the "Clear console" action for the console preview panel

### PR DESCRIPTION
**Pull request by ValentinH**
**What kind of change does this PR introduce?**
Add the "Clear console" action for the console preview panel.

**What is the new behavior?**
clear-console

**Checklist**:

- [ ] Documentation N/A
- [ ] Tests (didn't find existing one for this part of the code)
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
-    <Navigator
-      alignRight={alignRight}
-      alignBottom={alignBottom}
-      title="Console"
-    />
-    <Console.Content />
-  </div>
-);
+export default ({ alignRight, alignBottom }) => {
+  const actions = [
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js]: Much cleaner, nice!

Comment (CompuIves) [reply packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js]: I was thinking we could maybe reuse the action exported by `Console.js`? These ones: https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/Preview/DevTools/Console/index.js#L169. Maybe a simple concat.

Comment (ValentinH) [reply packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js]: Yes makes sense.

Comment (ValentinH) [reply packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js]: I didn't noticed this console module was already imported, much cleaner indeed :)

Comment (ValentinH) [reply packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js]: PR updated 😉

**Comment by CompuIves** [comment]
Again, perfect! Thank you very much for the swift and clean PRs @ValentinH, I really appreciate it!

**Pull request merged**